src/com/amazon/ion/benchmark/schema/ReparsedType.java

```diff
@@ -51,26 +61,46 @@ private void handleField(IonValue field) {
         switch (field.getFieldName()) {
             case KEYWORD_NAME:
             case KEYWORD_TYPE:
+            case KEYWORD_OCCURS:
                 return;
             default:
                 constraintMap.put(field.getFieldName(), toConstraint(field));
         }
     }
 
     /**
-     * Redefining the getIsl method to convert type definition to IonStruct format.
-     * @return an IonStruct which contains constraints in type definition.
+     * Processing field 'occurs' and return the occurrence value.
+     * @param constraintStruct represents the <VARIABLY_OCCURRING_TYPE_REFERENCE> in IonStruct format.
+     * @return an integer value which represents the required occurrence of the field/element.
      */
-    public IonStruct getIsl() {
-        return (IonStruct) type.getIsl();
+    public static int getOccurs(IonStruct constraintStruct) {
+        IonValue occursValue = constraintStruct.get(KEYWORD_OCCURS);
+        // The default value for occurs is specific to each constraint;
+        // For constraint Field, the default value of occurs is optional.
+        // For constraint ordered_elements, the default value of occurs is required.
+        if (occursValue == null) {
+            // TODO: Processing the default value of occurs referring to specific constraint.
+            return 1;
```

Comment:
Suggestion: 

`occurs` value can not be negative hence return a negative value(for example return `-1`) and then based on `fields` or `ordered_elements` constraint you can put the default value to either `0` or `1`.

Reply:
> Suggestion:
> 
> `occurs` value can not be negative hence return a negative value(for example return `-1`) and then based on `fields` or `ordered_elements` constraint you can put the default value to either `0` or `1`.

This TODO task was solved by the commit :[ Adds logic to process default value of constraint 'occurs'](https://github.com/amzn/ion-java-benchmark-cli/pull/36/commits/16d02b228db997b3b247a5dc4e95f32e4635b8e6).